### PR TITLE
Update all dependencies and cleanup

### PR DESCRIPTION
**Pull request by XhmikosR**
Might need a rebase after the other deps PRs are merged and also TS complains so feel free to push any fixes here.

TODO:

- [x] Update react-i18next too

### CODE REVIEW

**Comment by XhmikosR** [comment]
I'm unsure which branch to target because this will conflict for sure... Can we bring development and development-FTL to the same level? Otherwise this will need to wait until this happens.

**Comment by DL6ER** [comment]
@XhmikosR We renamed the former `development-FTL` to `development`. This caused merge conflict you might want to look at.

**Comment by XhmikosR** [comment]
@DL6ER should be OK now. I also pushed a few other patches. Please try this and I could use some help with the TS errors. :)

PS. There might be more unused deps/devDeps but better tackle those in parts

File: package.json [hunk 1]
```diff
@@ -4,71 +4,70 @@
...
+    "@coreui/icons": "0.4.1",
+    "@coreui/react": "^2.5.4",
+    "@fortawesome/fontawesome-free": "^5.12.0",
+    "@reduxjs/toolkit": "^1.1.0",
     "bootstrap-daterangepicker": "^3.0.5",
-    "chart.js": "^2.8.0",
-    "i18next": "^17.0.11",
-    "i18next-browser-languagedetector": "^3.0.3",
+    "chart.js": "^2.9.3",
+    "i18next": "^19.0.1",
```
Comment (DL6ER) [inline package.json]: You're updating `i18next` from version 17.0.11 to 19.0.1. This is quite a large jump and probably explains the TS error you're seeing. Check if the interface changed over the two major versions.

Comment (XhmikosR) [reply package.json]: Yeah I actually was about to downgrade it

**Comment by XhmikosR** [comment]
Can we get a final review here? I have #466 too which improves things further IMHO.

**Comment by XhmikosR** [comment]
Can we get this merged? It includes all dependabot PRs and flattens then deps tree *a lot*.

**Pull request merged**